src/components/about.js

```diff
@@ -0,0 +1,219 @@
+import React from "react";
+
+export const About = () => {
+    const elvesPic = "https://images.unsplash.com/photo-1479091751765-c137d6458be6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80";
```

Comment:
why direct url instead of bundled images?

Reply:
This may depend on the project , I understand bundled images are good practice. This would be in the case of loading images from a cms (with a fetch api of course)

Reply:
actually both approaches have their pros and cons, I just want to understand why you pick that way